### PR TITLE
docs(rxStatusColumn): Page objects to jsdoc3

### PR DESCRIPTION
**Pull request by Droogans**
### LGTMs
- [x] Dev LGTM
- [x] QE LGTM

### CODE REVIEW

File: src/components/rxStatusColumn/scripts/rxStatusColumn.page.js [hunk 1]
```diff
@@ -11,15 +11,60 @@ var classNameToStatus = function (iconClassName) {
...
+ *     },
+ *
+ *     row: function (rowIndex) {
+ *         var rowElement = this.tblServers.get(rowIndex);
+ *         return {
+ *             // The tests below focus heavily on this table row property
+ *             get status() { return rxStatusColumn.initialize(rowElement.$('[rx-status-column]')); },
+ *
+ *             // just for the sake of having another example present
+ *             get title() { return rowElement.$('td+td').getText(); }
```
Comment (CITguy) [inline src/components/rxStatusColumn/scripts/rxStatusColumn.page.js]: _(applies to all properties/functions in example)_ Please place body of functions on separate line from function definition.  Having inline-defined functions/properties feels like a run-on sentence and it reduces the readability of the code.

**Comment by jj4th** [comment]
QE-LGTM... though I agree with Ryan on the inline functions.

File: src/components/rxStatusColumn/scripts/rxStatusColumn.page.js [hunk 2]
```diff
@@ -11,15 +11,64 @@ var classNameToStatus = function (iconClassName) {
...
+ * {@link rxStatusColumn.colors}.
+ *
+ * All examples in this documentation will assume that you're using code similar to what's shown
+ * in the example below.
+ * @see rxSortableColumn
+ * @example
+ * var repeaterString = 'server in servers';
+ * var myTable = {
+ *     get rootElement() { return $('.demo-status-column-table'); },
+ *     get tblServers() { return this.rootElement.all(by.repeater(repeaterString)); },
```
Comment (CITguy) [inline src/components/rxStatusColumn/scripts/rxStatusColumn.page.js]: Please perform the same updates as done for the `row` property.

Comment (Droogans) [inline src/components/rxStatusColumn/scripts/rxStatusColumn.page.js]: Yep

**Comment by CITguy** [comment]
LGTM

**Pull request merged**